mappings/com/mojang/blaze3d/vertex/VertexFormat.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/unmapped/C_rnldvdpe com/mojang/blaze3d/vertex/VertexFormat
+	FIELD f_bwfkwhrn elementMap Lcom/google/common/collect/ImmutableMap;
+	FIELD f_hspiadgl vertexBuffer I
+	FIELD f_lfhcaqlu vertexArray I
+	FIELD f_magvnmca size I
+	FIELD f_qpqqznjz offsets Lit/unimi/dsi/fastutil/ints/IntList;
+	FIELD f_ssbbkgsr elementBuffer I
+	FIELD f_thnfzsnr elements Lcom/google/common/collect/ImmutableList;
+	METHOD <init> (Lcom/google/common/collect/ImmutableMap;)V
+		ARG 1 elementMap
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
+	METHOD m_aunpfmwi getIntegerSize ()I
+	METHOD m_brvxvqux clearAttribState ()V
+	METHOD m_etycqjdi setupAttribStateInternal ()V
+	METHOD m_fwfqeqmd getOrCreateVertexArray ()I
+	METHOD m_mocqgpdr setupAttribState ()V
+	METHOD m_nsxyklrn getShaderAttributes ()Lcom/google/common/collect/ImmutableList;
+	METHOD m_nutnaydi getOrCreateElementBuffer ()I
+	METHOD m_smhrgwhk clearAttribStateInternal ()V
```

Comment:
Do we use attrib as a shorthand, commonly? I think these should be expanded to attribute

Reply:
Follows OpenGL naming